release-notes.md

```diff
@@ -1,3 +1,13 @@
+## 29 September 2017
+
+### New features
+
+This release marks a major update of our [Placeholder](https://github.com/pelias/placeholder) service that is in charge of managing administrative area names. All sorts of queries that needs to understand the name of a country, city, county, neighbourhood, or any other administrative area are improved! Especially improved should be queries that use non-English languages. The full details of the code changes are [in the Placeholder pull request](https://github.com/pelias/placeholder/pull/49).
+
+### Bug fixes
+
+Over the last few days we have fixed [several](https://github.com/pelias/api/pull/1006) [regressions](https://github.com/pelias/api/pull/1003) that could cause bad responses from, and sometimes even crash our API server.
```

Comment:
"that could cause bad responses from, and sometimes even crash our API server" -> "that could cause bad responses from, and sometimes even crash, our API server" (adds a comma for proper phrasing)

Reply:
haha - good comma use, @trescube

Reply:
Updated for both fixes, thanks :)